zqd/api/connection.go

```diff
@@ -209,6 +209,23 @@ func (c *Connection) Search(ctx context.Context, search SearchRequest, params ma
 	return NewZngSearch(r), nil
 }
 
+func (c *Connection) IndexSearch(ctx context.Context, space SpaceID, search IndexSearchRequest, params map[string]string) (Search, error) {
+	req := c.Request(ctx).
+		SetBody(search).
+		SetQueryParam("format", "zng")
+
+	for n, v := range params {
+		req.SetQueryParam(n, v)
+	}
```

Comment:
SetQueryParams does the same thing: https://github.com/go-resty/resty/blob/v2.3.0/client.go#L258
```suggestion
		SetQueryParam("format", "zng").
		SetQueryParams(params)
```

Reply:
(we also at some point need to stop passing accept-type as a query parameter)